tendermint/src/block/signed_header.rs

```diff
@@ -13,23 +14,11 @@ pub struct SignedHeader {
     pub commit: block::Commit,
 }
 
-impl lite::SignedHeader for SignedHeader {
-    type Header = block::Header;
-    type Commit = SignedHeader;
-
-    fn header(&self) -> &block::Header {
-        &self.header
-    }
-
-    fn commit(&self) -> &Self {
-        &self
-    }
-}
-
 impl SignedHeader {
-    /// This is a private helper method to iterate over the underlying
+    /// This is a helper method to iterate over the underlying
     /// votes to compute the voting power (see `voting_power_in` below).
-    fn iter(&self) -> Vec<Option<SignedVote>> {
+    // TODO move into lite impl module?
+    pub fn iter(&self) -> Vec<Option<SignedVote>> {
```

Comment:
Similarly to https://github.com/interchainio/tendermint-rs/pull/124/files#r364984494
We probably won't use this somewhere else than in the `lite::Commit` impl for `SignedHeader`.